.vib/appsmith/cypress/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "username": "user@example.com",
```

Comment:
Keep in mind you are still using the default username. It would be good to update it to a non-default value and add it to the `runtime_params`.